frequency: 20
timeout_silence: 3
timeout_not_respond: 2 
regulators: ["thrust", "depth", "heading", "pitch", "position"]
params:
    depth:
        kp: 0.75
        ki: 0.0
        kd: 1.0
        accuracy: 0.15
        buoyancy_thrust: 0.045

    heading:
        kp: 0.13
        ki: 0.0
        kd: 2.5
        accuracy: 0.05

    pitch:
        kp: 0.25
        ki: 0.0
        kd: 12.5
        accuracy: 0.05

    position:
        dependencies: ["heading"]
        cruiser_dist: 5.0

        fwd_kp: 4.0
        fwd_ki: 0.0
        fwd_kd: 2.0

        side_kp: 4.0
        side_ki: 0.0
        side_kd: 2.0

        accuracy: 0.10


limits:
    forward: 0.5
    right: 0.3
    down: 0.7
    mforward: 0.3
    mright: 0.3
    mdown: 0.2
